[2.0.0-rc.8] tsickle version mis-match

## Type of Issue

```
[x] Bug Report
[ ] Feature Request
```

## Description
```bash
➜  core git:(master) ✗ npm run build:modules

> ng-packagr

module.js:538
    throw err;
    ^

Error: Cannot find module 'tsickle/src/tsickle'
    at Function.Module._resolveFilename (module.js:536:15)
    at Function.Module._load (module.js:466:25)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/***/packages/core/node_modules/ng-packagr/lib/util/ngc-patches.js:12:17)

```


#### How To Reproduce

just following the instructions...

#### Expected Behaviour

the file `node_modules/ng-packagr/lib/util/ngc-patches.js` contains the following:
```js
const tsickle = require("tsickle/src/tsickle");
```
if i change that manually to the following, everything works as expected
```js
const tsickle = require("tsickle");
```


#### Version Information

```
ng-packagr: 2.0.0-rc.7
@angular/*: 5.0.5
typescript: 2.4.2
rxjs: 5.5.2
node: 8.9.1
npm: 5.6.0
```

_please include any version information that might be relevant, e.g. other third-party libraries_

@dherges confirming, facing the same issue
```
$ ng-packagr
module.js:538
    throw err;
    ^
Error: Cannot find module 'tsickle/src/tsickle'
    at Function.Module._resolveFilename (module.js:536:15)
    at Function.Module._load (module.js:466:25)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

--- a/package.json
+++ b/package.json
@@ -50,7 +50,7 @@
   "peerDependencies": {
     "@angular/compiler": "^5.0.0",
     "@angular/compiler-cli": "^5.0.0",
-    "tsickle": ">=0.24.0 <0.26",
+    "tsickle": ">=0.25.5 <0.26",
     "typescript": ">=2.4.2 <2.6"
   },
   "devDependencies": {

